data/ships.txt

```diff
@@ -226,7 +224,8 @@ ship "Bactrian"
 	explode "large explosion" 45
 	explode "huge explosion" 30
 	"final explode" "final explosion large"
-	description "The Lionheart Bactrian is the last of the great city-ships, a design hearkening back to the early days of space colonization when a long-distance vessel needed to be a self-contained world, able to survive at times for weeks without encountering an inhabited planet. It is not only a freighter, but a carrier, and a capable warship either at short range or at a distance. Naturally, this versatility also makes it extremely expensive, and the Bactrian is not normally for sale to ordinary citizens who have not been vetted by the local government of the Deep."
+	description "The Lionheart Bactrian is the last of the great city-ships, a design hearkening back to the early days of space colonization when a long-distance vessel needed to be a self-contained world, able to survive at times for weeks without encountering an inhabited planet."
+	description "During the caravan era of the 27th century, the Bactrian was among the most powerful warships in human space, as well as being one of the best freighters and carriers. However, more modern times have seen the Bactrian become among the weakest of the warships, and it was not too many years ago that the Bactrian was recategorized as a transport vessel, and as such the Deep no longer requires for those who wish to purchase this massive ship to be vetted."
```

Comment:
> Gramatically, a sale cannot be 'vetted'--do you start to see the problem here?

Ok, didn't know that. Anyway, I didn't intend for my version to be the final one, just provided possible approach. Any one of you could rewrite it better. I'll just replace one word:

> During the caravan era of the 27th century, up-gunned Bactrians were among the most powerful warships in human space, and any sales were to be confirmed by local government of the Deep. No match for today's modern warships, the Bactrian is now classified as a Transport and sold freely to civilians.

Or:

> During the caravan era of the 27th century, up-gunned Bactrians were among the most powerful warships in human space, and all sales were controlled by local government of the Deep. No match for today's modern warships, the Bactrian is now classified as a Transport and sold freely to civilians.

Choose whatever is correct and fits best.